prog/dftb+/prg_dftb/initprogram.F90

```diff
@@ -2052,17 +2062,17 @@ subroutine initProgramVariables(input, env)
 
     tWriteChrgAscii = input%ctrl%tWriteChrgAscii
 
-    if (tReadChrg) then
+    !if (tReadChrg) then
```

Comment:
Any reason for commenting out these lines instead of removing? I think I see why they are redundant.